ocs_ci/framework/main.py

```diff
@@ -59,6 +78,7 @@ def main(arguments):
         '-p', 'ocs_ci.framework.pytest_customization.reports',
         '--logger-logsdir', pytest_logs_dir,
     ])
-    utils.add_path_to_env_path(os.path.expanduser(
-        framework.config.RUN['bin_dir']))
+    bin_dir = framework.config.RUN.get('bin_dir')
+    if bin_dir:
+        utils.add_path_to_env_path(bin_dir)
```

Comment:
Why do we have to add `bin_dir` to the path here when we are already adding the absolute path in `init_ocsci_conf`?

Reply:
We always added bin dir in the PATH. So didn't changed the logic here. Just moved it all to the function init config for now to have it in one place.

Reply:
@petr-balogh right, we were doing it in two places when I had commented this.